features/asia/india/india_full.geojson

```diff
@@ -0,0 +1,148 @@
+{
+  "type": "Feature",
+  "id": "india_full",
+  "properties": {"name": "India"},
+  "geometry": {
+    "type": "MultiPolygon",
+    "coordinates": [
```

Comment:
looks like MultiPolygon needs an additional nesting of coordinate array rings (add `[` )